samples/ABNF/tranquil.abnf

```diff
@@ -0,0 +1,61 @@
+; Source:  https://github.com/fjolnir/Tranquil
+; License: BSD 3 Clause License
+; Modified for standard compatibility by René Schwaiger
```

Comment:
The [original file](https://github.com/fjolnir/Tranquil/blob/master/Docs/UnusedOld/tranquil.abnf) uses single quotes, which are not part of the [standard](https://tools.ietf.org/pdf/rfc5234.pdf) as far as I can tell.